### PR TITLE
Fix Narrator not announcing on delete at the last char

### PR DESCRIPTION
**Pull request by BryanValverdeU**
Issue:
The Narrator is not announcing the deleted charater when on the last element.

Before the change: (Notice how the narrator is not announcing the deleted charater)
Sppech on deleting B4

After
Sppech on deleting after

Code Changes:

Added a function to announce the deleted charater, only when is at the end of the line.
Also a new content edit feature to announce the message to the narrator.

This seems to be a browser issue. I tried to repro the behavior and I was able to repro an a bare HTML Div contenteditable.

### CODE REVIEW

File: packages/roosterjs-editor-dom/lib/utils/narratorMessage.ts [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import { ContentPosition, IEditor } from 'roosterjs-editor-types';
+
+const MESSAGE_CONTAINER_ID = 'roosterjs_NarratorMessage';
+let MESSAGE_JOB: number;
+/**
+ * Makes the narrator announce a message
+ * @param file The file to read
```
Comment (Lufedi) [inline packages/roosterjs-editor-dom/lib/utils/narratorMessage.ts:7]: wrong parameter

File: packages/roosterjs-editor-dom/lib/utils/narratorMessage.ts [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    }
+
+    container.appendChild(el);
+
+    MESSAGE_JOB = currentWindow.setTimeout(function () {
+        editorDocument.getElementById(id).textContent = message;
+    }, 200);
+
+    currentWindow.setTimeout(function () {
+        const element = editorDocument.getElementById(id);
```
Comment (Lufedi) [inline packages/roosterjs-editor-dom/lib/utils/narratorMessage.ts:50]: Is it possible to move this to the  MESSAGE_JOB setTimeOut? or why do we need to have two ´setTimeouts´ ?

File: packages/roosterjs-editor-types/lib/interface/ContentEditFeatureSettings.ts [hunk 3]
```diff
@@ -201,6 +201,33 @@ export interface TableFeatureSettings {
     upDownInTable: boolean;
 }
 
+/**
+ * Settings for table features
+ */
+export interface TableFeatureSettings {
```
Comment (Lufedi) [inline packages/roosterjs-editor-types/lib/interface/ContentEditFeatureSettings.ts:207]: I this this is not part of this PR

**Comment by BryanValverdeU** [comment]
Seems like this is an error in Browser. Closing this PR for now.
